AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -88,10 +88,11 @@ def test_ShapeCollider_CylinderShapeCollides(self, request, workspace, editor, l
         from .tests.shape_collider import ShapeCollider_CylinderShapeCollides as test_module
         self._run_test(request, workspace, editor, test_module)
 
+    @pytest.mark.xfail(reason="LYN-10733 - Error messages from deleting entities")
```

Comment:
If the test is fixed and passing, why does it need to be marked as xfail?

Reply:
+1 - also looks like this is related to this GHI - https://github.com/o3de/o3de/issues/7946

Something @srikappa-amzn is looking at? Should this be closed or is there a deeper problem yet to be solved?

Reply:
Yes I'm looking into https://github.com/o3de/o3de/issues/7946 
@chiyenteng said that he'll be waiting on that fix to test and merge this then.